package.json

```diff
@@ -17,14 +17,21 @@
   },
   "devDependencies": {
     "@kadira/storybook": "^2.21.0",
+    "@kadira/storybook-deployer": "^1.2.0",
     "react-scripts-ts": "^1.4.0",
     "styled-components": "^1.4.6",
     "typescript": "^2.3.2"
   },
   "scripts": {
-    "build": "./node_modules/.bin/tsc",
+    "build": "tsc",
     "test": "react-scripts-ts test --env=jsdom",
     "storybook": "start-storybook -p 6006 -s ./stories/assets",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook -s ./stories/assets",
+    "deploy-storybook": "storybook-to-ghpages"
+  },
+  "storybook-deployer": {
```

Comment:
Do we need `storybook-deployer`? According to the docs it's the default: https://github.com/storybooks/storybook-deployer